### ISSUE TITLE
request for ktensor viz functionaliry

### ISSUE THREAD

**Issue by tgkolda** [sandialabs/pyttb#365]
I hope it will be possible to add the ktensor viz functionality. Happy to have a discussion about it. It's pretty critical for the book. I hadn't realized that it's not included yet.

**Comment by ntjohnson1** [comment]
Hmm, it looks like when it was added it was added as `vis` not `viz`. #304 and #301

@ghbrown was the difference in naming intentional? I don't see a discussion on it in the issue or PR, so I assume we should make it consistent if there wasn't a reason for it.

**Comment by ghbrown** [comment]
> was the difference in naming intentional?

Not really. While my personal preference is `vis`, I just forgot that the interface should be similar to TTB when I was writing the PR.

I have no problems if it's renamed, I can make the PR if you'd like.

**Comment by ntjohnson1** [comment]
No worries. I assumed that was the case but just wanted to confirm. You're welcome to open the PR (but Danny won't be able to approve for a bit) or I'm happy to handle it with some other cleanup this month.

**Comment by tgkolda** [comment]
Ah - thanks for the reminder. I saw a note that viz was not copied over here: https://pyttb.readthedocs.io/en/stable/matlab/ktensor.html#matlab-methods-not-included-in-pyttb. 

Stupis question which maybe should be a discussion or separate request... How should I have found the vis command without already knowing its spelling? In MATLAB, every class has a summary of all its methods that you can get by typing, e.g., `help ktensor`. Is there an equivalent here? I know I can scroll though all the methods in the documentation, but I'm wondering if there's a list somewhere with one line descriptions. Thanks.

**Comment by ntjohnson1** [comment]
Ya, our MATLAB docs are just raw text at the moment so we can't programmatically verify them. Can fix that when we rename viz. Lingering todo to make the MATLAB docs have code so we can automatically detect if things go stale.

There are a few options in python. Here are two from the command line. Alternatively most IDEs should provide tab complete or tab to list methods for an existing object.
```python
>>>import pyttb as ttb
# For full documentation (a little verbose)
>>>help(ttb.ktensor)
<snip printout>
>>>import inspect
# To just get the signatures and names
>>>inspect.getmembers(ttb.ktensor)
<snip printout>
```

**Comment by dmdunla** [comment]
Closed by 589239b28d635e4619e4ac208281b537622af1fb

Name has been updated.
